oauth-common/src/main/java/io/strimzi/kafka/oauth/validator/TokenValidator.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright 2017-2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.oauth.validator;
+
+import io.strimzi.kafka.oauth.common.TokenInfo;
+
+public interface TokenValidator {
+
+    TokenInfo validate(String token);
```

Comment:
A little javadoc about the contract this is defining (e.g. the exceptions thrown when the token is invalid) probably wouldn't hurt.

Reply:
Yeah, I need to put more JavaDoc everywhere.